admin/app/com/lucidchart/piezo/admin/models/MonitoringTeams.scala

```diff
@@ -0,0 +1,28 @@
+package com.lucidchart.piezo.admin.models
+
+import play.api.Configuration
+import java.nio.file.Files
+import play.api.libs.json.Json
+import scala.util.Try
+import java.io.File
+import java.io.FileInputStream
+import play.api.libs.json.JsArray
+
+class MonitoringTeams(configuration: Configuration) {
```

Comment:
Another nit. I would change this to take a `Seq[String]` as an input, and have the configuration parsing in a separate constructor, or a constructor method on the companion object.

ex:

```scala
def this(configuration: Configuration) = MonitoringTeams({
   // calculate teams from config here
})
```